variantlib/commands/make_variant.py

```diff
@@ -130,19 +130,25 @@ def make_variant(args: list[str]) -> None:
     pyproj_toml = VariantPyProjectToml(toml_data={})
 
     with contextlib.suppress(AttributeError):
-        pyproj_toml.namespace_priorities = parsed_args.default_namespace_prio.split(",")
+        pyproj_toml.namespace_priorities = (
+            parsed_args.default_namespace_priorities.split(",")
+        )
         for ns in pyproj_toml.namespace_priorities:
             if VALIDATION_NAMESPACE_REGEX.fullmatch(ns) is None:
                 raise ValueError(f"The following namespace is invalid: `{ns}`")
     with contextlib.suppress(AttributeError):
-        pyproj_toml.feature_priorities = parsed_args.default_feature_prio.split(",")
+        pyproj_toml.feature_priorities = parsed_args.default_feature_priorities.split(
+            ","
+        )
         for vfeat in pyproj_toml.feature_priorities:
-            if VALIDATION_FEATURE_REGEX.fullmatch(vfeat) is None:
+            if VALIDATION_FEATURE_REGEX.fullmatch(vfeat.to_str()) is None:
                 raise ValueError(f"The following feature is invalid: `{vfeat}`")
     with contextlib.suppress(AttributeError):
-        pyproj_toml.property_priorities = parsed_args.default_property_prio.split(",")
+        pyproj_toml.property_priorities = parsed_args.default_property_priorities.split(
+            ","
+        )
         for vprop in pyproj_toml.property_priorities:
-            if VALIDATION_PROPERTY_REGEX.fullmatch(vprop) is None:
+            if VALIDATION_PROPERTY_REGEX.fullmatch(vprop.to_str()) is None:
```

Comment:
Same - Did you test that ? vprop is a string

Reply:
Fixed now.